ion-schema/src/violation.rs

```diff
@@ -146,3 +150,336 @@ impl fmt::Display for ViolationCode {
         )
     }
 }
+
+#[macro_export]
+macro_rules! assert_equivalent_violations {
```

Comment:
Please add a brief doc comment for this and for `assert_non_equivalent_violations`.